File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        i = 0
        j = len(s)-1
        
        while i < j:
          front = s[i]
          back = s[j]
          s[i] = back
          s[j] = front
          i += 1
          j -= 1

File: 0344-reverse-string/0344-reverse-string.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
  let i = 0, j = s.length - 1;
  while (i < j) {
    let front = s[i], back = s[j]
    s[i] = back;
    s[j] = front;
    i++;
    j--;
  }
};